apps/admin-gui/src/app/vos/pages/member-detail-page/member-overview/member-overview.component.ts

```diff
@@ -37,26 +39,51 @@ export class MemberOverviewComponent implements OnInit {
   member: RichMember = null;
   navItems: MenuItem[] = [];
 
+  attributeNames: Array<string> = [];
+  attributes: Map<string, string[]> = new Map<string, string[]>();
+  dataSource = new MatTableDataSource<string>();
+  membershipDataSource = new MatTableDataSource<string>();
+  displayedColumns = ['attName', 'attValue'];
+  sponsors: RichUser[] = [];
+  sponsorsDataSource = new MatTableDataSource<string>();
+
   vo: Vo;
   loading = false;
 
   ngOnInit() {
     this.loading = true;
     this.route.parent.params.subscribe(parentParams => {
       const memberId = parentParams['memberId'];
+      this.attributeNames = this.storeService.getMemberProfileAttributeNames();
 
       this.membersService.getRichMemberWithAttributes(memberId).subscribe(member => {
         this.member = member;
         this.fullName = parseFullName(this.member.user);
         this.statusIcon = parseStatusIcon(this.member);
         this.statusIconColor = parseStatusColor(this.member);
 
+        this.initAttributes();
+        this.dataSource = new MatTableDataSource<string>(Array.from(this.attributes.keys()));
+        this.membershipDataSource = new MatTableDataSource<string>(['Status', 'Expiration']);
+        this.sponsorsDataSource = new MatTableDataSource<string>(['Sponsors']);
+
         this.vo = {
           id: member.voId,
           beanName: "Vo"
         };
-        this.initNavItems();
-        this.refreshData();
+
+        if (this.member.sponsored) {
+          this.membersService.getSponsorsByMember(this.member.id, null).subscribe(sponsors => {
```

Comment:
because of that I would label the card for this "This member is sponsored" so that at least group admin know that it is sponsored.
The rest of the card would appear only if the user have priviledge to call this method. In the card content would be "Sponsors:" and under that table with the names of the sponsors, one name on one row.